### DISCUSSION TITLE
PhotoPrismPi - Raspberry setup

### DISCUSSION THREAD

**Discussion by dartoi** [photoprism/photoprism#3290] [Q&A] [answered]
Dear All, 
I have put on SD Card through balenaEtcher the image "photoprismpi-latest-22.04-preinstalled-server-arm64+raspi-220715.img", the I have link Raspberry with wired connection and switch on it. 

After some minutes, I have tried to link http://photoprismpi.local/ or IP (that I see on my router panel) but not working. I used the IP on PuTTy and it's ok, but I not see the web interface. 

Why? 
Thank for your support.

**Comment by lastzero** [comment]
There is a possibility that this is broken due to changes in the Docker packages, as we recently had the same issue on our production servers. Unfortunately, we are traveling until Sunday... It might help to manually install the compose plugin:

https://docs.photoprism.app/getting-started/troubleshooting/docker/#docker-compose

**Reply by lastzero** [reply]
We are currently building an updated version which will hopefully work for you. Looks like it may indeed have been an issue with Docker Compose, as the legacy "docker-compose" command was replaced by "docker compose".

**Reply by lastzero** [reply]
:point_right: https://dl.photoprism.app/dist/photoprismpi/

**Comment by lastzero** [comment, marked as answer]
I have verified that today's build works properly if you follow our docs:

- https://docs.photoprism.app/getting-started/raspberry-pi/microsd-image/

The problem was the change from the old `docker-compose` command to the compose plugin that you run with `docker compose`:

- https://docs.photoprism.app/getting-started/troubleshooting/docker/#docker-compose

**Note that the latest PhotoPrism version is downloaded by the Raspberry Pi image when you start it. So it requires a working internet connection and you may have to wait about 10 - 15 minutes before you can connect with a browser.**
